### DISCUSSION TITLE
Recalculate model boundary after animation started

### DISCUSSION THREAD

**Discussion by arthurrmp** [google/model-viewer#4453] [Q&A]
Some of the models I'm working with have a default pose where objects are positioned under their feet, as shown in this image:

![image](https://github.com/google/model-viewer/assets/44805300/03aeab6c-c230-4a1a-a405-2fa1a19ddea0)

This results in the models appearing to float when any animation is played, as demonstrated in the second image:

![image](https://github.com/google/model-viewer/assets/44805300/d8761244-0e3e-4088-a236-7201909f8ed9)

Probably this is a problem of the models, but unfortunately, manually fixing each model is not a feasible option.

So, I've been thinking if there might be an alternative solution to address this problem. Is there a possible workaround to recalculate the bounds after the animation has started? 

Example: https://codesandbox.io/s/boring-currying-fylslx

**Comment by elalish** [comment]
If you have a fairly general solution, PRs are always welcome. Otherwise, I'm curious if this (vague) proposal would be adequate for your use case? #4442.
